apps/src/turtle/ArtistVisualizationColumn.jsx

```diff
@@ -13,10 +15,12 @@ var styles = {
 };
 
 var ArtistVisualizationColumn = function(props) {
+  const showSaveImageButton = !!appOptions.level.enableDownloadImage;
```

Comment:
this should probably be a component prop instead of reaching into appOptions. it looks like artist.js already consumes some level configuration properties:
https://github.com/code-dot-org/code-dot-org/blob/8a13e33c92c534ed295fe43130e7e3f4da5483cf/apps/src/turtle/artist.js#L346-L349